package.json

```diff
@@ -22,9 +22,10 @@
     "test:cucumber-super-admin": "cucumber-js --retry 5 --parallel 8 --require-module 'ts-node/register' --require 'src/test/functional/**/*.ts' --tags 'not @skipped and not @pending' src/test/functional/features/super_admin/*.feature",
     "test:cucumber-admin": "cucumber-js --retry 3 --parallel 3 --require-module 'ts-node/register' --require 'src/test/functional/**/*.ts' --tags 'not @skipped and not @pending' src/test/functional/features/admin/*.feature",
     "test:functional1": "yarn test:cucumber-viewer && yarn test:cucumber-super-admin && yarn test:cucumber-admin && yarn test:pa11y",
-    "test:functional": "npm-run-all -scl test:codecept test:functional:report test:pa11y",
+    "test:functional": "npm-run-all -scl test:codecept test:codecept:report test:pa11y",
     "test:functional:report": "allure generate functional-output/functional/reports/ -c -o functional-output/functional/allure",
-    "test:codecept": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs run-workers 4 parallel --config ./src/test/functional/codecept.conf.js --grep @special",
+    "test:codecept": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs run-workers 8 parallel --config ./src/test/functional/codecept.conf.js",
```

Comment:
Once all the other changes are made can you test runs of the workers:

- 10 workers
- 8 workers
- 6 workers
- 4 workers
- 2 workers

and collect the allure reports for each so we can compare